codegen-server/python/src/main/kotlin/software/amazon/smithy/rust/codegen/server/python/smithy/PythonServerCodegenVisitor.kt

```diff
@@ -61,14 +61,43 @@ class PythonServerCodegenVisitor(
             )
                 .protocolFor(context.model, service)
         protocolGeneratorFactory = generator
+
         model = codegenDecorator.transformModel(service, baseModel)
-        symbolProvider = PythonCodegenServerPlugin.baseSymbolProvider(model, service, symbolVisitorConfig)
 
-        // Override `codegenContext` which carries the symbolProvider.
-        codegenContext = ServerCodegenContext(model, symbolProvider, service, protocol, settings)
+        // `publicConstrainedTypes` must always be `false` for the Python server.
```

Comment:
I think I understand, but care to explain in the comment why it must be always false?